testdata/p4_16_errors_outputs/functors2_e.p4-stderr

```diff
@@ -1,3 +1,6 @@
-functors2_e.p4(20): [--Werror=type-error] error: p: Cannot refer to control inside itself
+functors2_e.p4(20): [--Werror=invalid] error: Recursive declaration of 'p' in 'control p'
```

Comment:
This message is not accurate, since this is not a declaration. The previous message was more precise.

Reply:
I very much agree, which is why I've been meaning to ask: is there a more suitable error message (one I can use in resolveReferences.cpp) or is adding a new error acceptable?

Reply:
The main goal is to make the life of the user easy; if adding a new message is the best way, it should be added.